saltproc/saltproc.py

```diff
@@ -332,12 +332,19 @@ def write_run_info(self):
                 cat = 'atomic'
                 dens_dict[key] = float(value.split()[2])
         for key, value in dens_dict.items():
-            self.f.create_dataset('siminfo_%s_%s_density' %
-                                  (key, cat), data=value)
-        # init composition
+            if key == self.driver_mat_name:
+                key = 'driver'
+            elif key == self.blanket_mat_name:
+                key = 'blanket'
+            self.f.create_dataset('siminfo_%s_%s_density' %(key, cat), data=value)
+
         init_comp = self.read_dep(boc=True)
         for key, value in init_comp.items():
-            self.f.create_dataset('siminfo_%s_init_comp' % key, data=value)
+            if key == self.driver_mat_name:
+                key = 'driver'
+            elif key == self.blanket_mat_name:
+                key = 'blanket'
```

Comment:
Why are these four lines copy pasted from above?  Maybe they should be their own function.

Never copy paste. Use the DRY principle (Don't Repeat Yourself)